Twilio-org/phonebank
# Campaign User Association Endpoint

Description:
- Create getCampaignUserAssociation service and controller
- Create the /users/:id/campaigns/:campaign_id route endpoint
- Test the getCampaignUserAssociation service

Test:
- run `npm run test:server`
- Manual:
  - Comment out passport authentication in user routes file
  - run `npm run build` and `npm run start`
  - Use postman to send a get request to `http://localhost:3000/users/1/campaigns/1`
    - should return a 204 with no body (association exists)
  - Use postman to send a get request to `http://localhost:3000/users/1/campaigns/100`
    - should return a 404 with no body (association doesn't exist)
 - Use postman to send a get request to `http://localhost:3000/users/1000/campaigns/1`
    - should return a 404 with no body (association doesn't exist)

Jira Ticket: 98
